### DISCUSSION TITLE
Emoji is displayed within the quoting code

### DISCUSSION THREAD

**Discussion by kumavale** [community/community#65755] [Repositories]
### Select Topic Area

Bug

### Body

I committed with a message like below.

```
refactor: use `tokio::task::spawn_bloking()` instead of `std::thread::spawn()`
```

Then, conversion to unintended emoji is performed in the Quoting code (.e.g, `:thread:` to `🧵`)

<kbd>![image](https://github.com/community/community/assets/29778890/9537445c-f1e9-49b3-bce9-59e4b99fd30a)</kbd>

**Comment by KiGamji** [comment]
just found out that the same thing happens in links although they're opening fine: #68135
